### ISSUE TITLE
[CoreOS] Failed to init driver: px/flexvolume

### ISSUE THREAD

**Issue by maikotz** [portworx/px-dev#17]
Hi, 

I wanted to take a look into px-dev and could set it up on my dev-master node but the other node is failing.

```
Jan 31 11:22:37 a-10-10-23-86 kubelet-wrapper[10550]: E0131 11:22:37.679792   10550 flexvolume_util.go:98] Failed to init driver: px/flexvolume, error: exit status 1
Jan 31 11:22:37 a-10-10-23-86 kubelet-wrapper[10550]: E0131 11:22:37.679826   10550 flexvolume_util.go:79] Failed to unmarshal output for command: init, output: 2017-01-31T11:22:37Z NONE  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 0.0.0.0:9005: getsockopt: connection refused"; Reconnecting to {0.0.0.0:9005 <nil>}
Jan 31 11:22:37 a-10-10-23-86 kubelet-wrapper[10550]: rpc error: code = 14 desc = grpc: the connection is unavailable
Jan 31 11:22:37 a-10-10-23-86 kubelet-wrapper[10550]: , error: invalid character '-' after top-level value
Jan 31 11:22:37 a-10-10-23-86 kubelet-wrapper[10550]: E0131 11:22:37.679851   10550 plugins.go:339] Failed to load volume plugin &{px/flexvolume /etc/kubernetes/kubelet-plugins/volume/exec/px~flexvolume %!s(*kubelet.kubeletVolumeHost=&{0xc4204da000 {{1 0} map[kubernetes.io/downward-api:0xc4202f6610 kubernetes.io/empty-dir:0xc4202f60e0 kubernetes.io/secret:0xc4202f6310 kubernetes.io/nfs:0xc420c7a190 kubernetes.io/flocker:0xc4202f6630 kubernetes.io/fc:0xc420cdf300 kubernetes.io/git-repo:0xc4202f62f0 kubernetes.io/host-path:0xc420c7a140 kubernetes.io/iscsi:0xc420cdf280 kubernetes.io/glusterfs:0xc420cdf2a0 kubernetes.io/rbd:0xc420cdf2c0 kubernetes.io/cinder:0xc420cdf2e0 kubernetes.io/aws-ebs:0xc4202f6010 kubernetes.io/gce-pd:0xc4202f62e0 kubernetes.io/quobyte:0xc4202f6320 kubernetes.io/cephfs:0xc4202f65d0]}})}, error: invalid character '-' after top-level value
Jan 31 11:22:37 a-10-10-23-86 kubelet-wrapper[10550]: Error: failed to run Kubelet: failed to create kubelet: Could not initialize volume plugins for KubeletVolumePluginMgr: invalid character '-' after top-level value
Jan 31 11:22:37 a-10-10-23-86 kubelet-wrapper[10550]: failed to run Kubelet: failed to create kubelet: Could not initialize volume plugins for KubeletVolumePluginMgr: invalid character '-' after top-level value
```

I'm using the same config as on my master node with a loopback device for testing purposes. 

PX Container is running and has no error's in the logs.

```
sudo /opt/pwx/bin/pxctl status
Status: PX is operational
Node ID: 15833480-8141-457d-8fda-a5a6023fc192
	IP: 10.10.23.86
 	Local Storage Pool: 1 pool
	Pool	IO_Priority	Size	Used	Status	Zone	Region
	0	MEDIUM		37 GiB	2.0 GiB	Online	default	default
	Local Storage Devices: 1 device
	Device	Path		Media Type		Size		Last-Scan
	0:1	/dev/loop0	STORAGE_MEDIUM_MAGNETIC	37 GiB		31 Jan 17 11:31 UTC
	total			-			37 GiB
Cluster Summary
	Cluster ID: 5ac2ed6f-7e4e-4e1d-8e8c-3a6df1fb61a7
	Node IP: 10.10.23.86 - Capacity: 2.0 GiB/37 GiB Online (This node)
	Node IP: 10.10.23.10 - Capacity: 2.0 GiB/40 GiB Online
Global Storage Pool
	Total Used    	:  4.0 GiB
	Total Capacity	:  77 GiB
```

Kubelet is v1.5.1_coreos.0

**Edit:**
I think the key here is the connection refused on :9005

If I try to run the flexvolume init manually I get the same message:
```
$ /etc/kubernetes/kubelet-plugins/volume/exec/px~flexvolume/flexvolume init
2017-01-31T12:10:51Z NONE  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 0.0.0.0:9005: getsockopt: connection refused"; Reconnecting to {0.0.0.0:9005 <nil>}
rpc error: code = 14 desc = grpc: the connection is unavailable
```

**Comment by maikotz** [comment]
This issue was resolved by downgrading from ``latest`` tag to 1.0.9

**Comment by adityadani** [comment]
This issue has also been resolved in the latest px-dev build.
You can download it from portworx/px-dev:latest
